lib/elements/element.py

```diff
@@ -198,6 +209,15 @@ def path(self):
         # Tuples all the way down.  Hasn't anyone heard of using classes?
 
         d = self.node.get("d", "")
+
+        if not d:
```

Comment:
It's possible someone manually set a `d` on a rectangle.  It's invalid for an SVG and Inkscape would ignore it (I think), but the result would be very confusing for an Ink/Stitch user.  How about only fetch the `d` tag if it's not a rectangle, ellipse, or circle?

Reply:
Ok, yes. That makes sense.